### PR TITLE
v5.13.0

### PR DESCRIPTION
**Pull request by flaviendelangle**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -3,6 +3,56 @@
 All notable changes to this project will be documented in this file.
 See [Conventional Commits](https://conventionalcommits.org) for commit guidelines.
 
+## 5.13.0
+
+_Jul 7, 2022_
+
+We'd like to offer a big thanks to the 9 contributors who made this release possible. Here are some highlights ✨:
+
+- 🎁 `@mui/x-date-pickers` and `@mui/x-date-pickers-pro` are moving to beta
```
Comment (flaviendelangle) [inline CHANGELOG.md:12]: @joserodolfofreitas not sure how to present that change

File: CHANGELOG.md [hunk 2]
```diff
@@ -3,6 +3,56 @@
...
+
+- ✨ Allow to infer detail panel height from content (#5163) @m4theushw
```
Comment (flaviendelangle) [inline CHANGELOG.md:14]: @m4theushw if you want to improve the example

File: CHANGELOG.md (hunk 1 above)
Comment (DanailH) [reply CHANGELOG.md:12]:
```suggestion
- 🎁 `@mui/x-date-pickers` and `@mui/x-date-pickers-pro` are now in beta!
```
?

File: CHANGELOG.md (hunk 2 above)
Comment (DanailH) [reply CHANGELOG.md:14]: Isn't it more readable if we just say - Allow dynamic height for detail panel? Also, I think Matheus is on PTO until Friday.

File: CHANGELOG.md [hunk 3]
```diff
@@ -3,6 +3,59 @@
...
+
+  ```tsx
+  <DataGridPro getDetailPanelHeight={() => 'auto'} />
+  ```
+
+- 🌍 Add Romanian (ro-RO) locale on the data grid (#5345) @rolule
+- 📚 Documentation improvements
+- 🐞 Bugfixes
+
+-
```
Comment (DanailH) [inline CHANGELOG.md:24]:
```suggestion
```

**Review by alexfauquette — APPROVED** [review]

**Review by DanailH — APPROVED** [review]

**Pull request merged**